src/website/views/components/toolbar/_examples.pug

```diff
@@ -1,499 +1,1371 @@
-                </button>
-                <button class="chi-button -icon -flat" id="button-br-column-customization" aria-label="Column Customization" data-tooltip="Column Customization" data-position="top">
+                  <button class="chi-button -portal -icon -primary -flat" id="button-p-filters" aria-label="Filters" data-tooltip="Filters" data-position="top">
+                    <div class="chi-button__content">
+                      <i class="chi-icon icon-filter"></i>
+                    </div>
+                  </button>
+                </div>
+                <div class="chi-toolbar__filters-mobile">
+                  <button class="chi-button -portal -icon -primary -flat chi-drawer__trigger" id="drawer-trigger-bp1" data-target="#drawer-1" aria-label="Button action">
+                    <div class="chi-button__content">
+                      <i class="chi-icon icon-filter"></i>
+                    </div>
+                  </button>
+                </div>
+              </div>
+            </div>
+            <div class="chi-toolbar__end">
+              <div class="chi-toolbar__actions">
+                <div class="chi-toolbar__actions-desktop">
+                  <button class="chi-button -portal -icon -primary -flat" id="button-p-download" aria-label="Download" data-tooltip="Download" data-position="top">
+                    <div class="chi-button__content">
+                      <i class="chi-icon icon-arrow-to-bottom"></i>
+                    </div>
+                  </button>
+                  <button class="chi-button -portal -icon -primary -flat" id="button-p-refresh" aria-label="Refresh" data-tooltip="Refresh" data-position="top">
+                    <div class="chi-button__content">
+                      <i class="chi-icon icon-refresh"></i>
+                    </div>
+                  </button>
+                  <button class="chi-button -portal -icon -primary -flat" id="button-p-column-customization" aria-label="Column Customization" data-tooltip="Column Customization" data-position="top">
+                    <div class="chi-button__content">
+                      <i class="chi-icon icon-table-column-settings"></i>
+                    </div>
+                  </button>
+                </div>
+                <div class="chi-toolbar__actions-mobile">
+                  <button class="chi-button -portal -icon -primary -flat chi-drawer__trigger" id="drawer-trigger-bp2" data-target="#drawer-2" aria-label="Button action">
+                    <div class="chi-button__content">
+                      <i class="chi-icon icon-more-vert"></i>
+                    </div>
+                  </button>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+
+        <!-- Drawer -->
+        <div class="chi-backdrop -closed">
+          <div class="chi-backdrop__wrapper">
+            <div class="chi-drawer -left -portal -menu -position--absolute" id="drawer-1">
```

Comment:
-portal class is not needed anymmore, please remove

Reply:
Solved.